specification/cognitiveservices/OpenAI.Inference/models/images.tsp

```diff
@@ -0,0 +1,122 @@
+import "@azure-tools/typespec-azure-core";
+import "@typespec/http";
+
+using Azure.Core;
+using TypeSpec.Http;
+
+namespace Azure.OpenAI;
+
+@doc("The desired size of the generated images. Must be one of 256x256, 512x512, or 1024x1024.")
+enum ImageSize {
+    `256x256`,
+    `512x512`,
+    `1024x1024`,
+}
+
+@doc("Defines the request to create an operation to generate images.")
```

Comment:
This could use some extra simplification/wordsmithing for user-facing docs clarity. "Represents the request data used to generate images" or similar?

Reply:
updated